test/job_hooks_test.rb

```diff
@@ -342,6 +342,7 @@ def self.perform(history)
     Resque.dequeue(BeforeDequeueJob, history)
     @worker.work(0)
     assert_equal history, [:before_dequeue], "before_dequeue was not run"
+    assert_equal [:before_dequeue], history, "before_dequeue was not run"
```

Comment:
@spraints the method signature expects the expected value first and actual value second. Just helps when there is a failure.